lib/packwerk/parsed_constant_definitions.rb

```diff
@@ -38,16 +38,20 @@ def self.reference_qualifications(constant_name, namespace_path:)
     private
 
     def collect_local_definitions_from_root(node, current_namespace_path = [])
-      if Node.type(node) == Node::CONSTANT_ASSIGNMENT
-        add_definition(Node.constant_name(node), current_namespace_path, Node.name_location(node))
-      elsif Node.module_name_from_definition(node)
-        # handle compact constant nesting (e.g. "module Sales::Order")
-        tempnode = node
-        while (tempnode = Node.each_child(tempnode).find { |n| Node.type(n) == Node::CONSTANT })
-          add_definition(Node.constant_name(tempnode), current_namespace_path, Node.name_location(tempnode))
+      begin
```

Comment:
From what I can tell, you are only adding a rescue block here, correct?
In that case, I would suggest getting rid of the `begin` statement in this method. 
This is a good example of what I mean: http://seejohncode.com/2012/04/17/short-tip-rescuing-a-method-in-ruby/

Reply:
Yes!